build/common.props

```diff
@@ -13,9 +13,12 @@
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>https://github.com/auth0/auth0.net</RepositoryUrl>
     <PackageReleaseNotes>
-	  Version 7.0.9
-	  - Fix boolean casing on form post operations such as ImportUsersAsync so that
-	    upsert and sendCompletionEmail work.
+      Version 7.1.0
+      - Add support for Log Streams API in Auth0.ManagementApi
+      
+	    Version 7.0.9
```

Comment:
Formatting looks off?

Reply:
Yes that's weird, looked ok locally in Visual Studio. Looks like a spaces/tabs thing.

Fixed in the browser.